dashboard/app/app.yaml

```diff
@@ -34,3 +34,12 @@ handlers:
 - url: /(|api|bug|text|x/.+|.*)
   script: auto
   secure: always
+
+readiness_check:
+  # https://cloud.google.com/appengine/docs/flexible/custom-runtimes/configuring-your-app-with-app-yaml#readiness_checks
```

Comment:
Does this present in the standard environment?
I think we are using Standard Go1.11 environment, not the Flexible environment:
https://cloud.google.com/appengine/docs/go
I don't see this setting in the Standard environment reference:
https://cloud.google.com/appengine/docs/standard/go111/config/appref

Reply:
You're right... https://cloud.google.com/appengine/docs/flexible/go/flexible-for-standard-users :

Standard environment does not use health checks to determine whether or not to send traffic to an instance. Flexible environment permits application developers to write their own health check handlers that will be used by the load balancer to determine whether or not to send traffic to an instance and whether or not it should be autohealed. Developers should be careful when adding logic to health checks. For example, if the health check makes a call to an external service then a temporary failure in that service can cause all instances to go unhealthy, possibly leading to a cascading failure.